reduct-nepal/seep
# Insted of showing the accuracy only ,the user can also view thier tra…

Tasks completed:

- [x] allow user to see their transcript data and diff when they give practice Test.

### To Test:

- Login into the seep portal and take a practice Test
- Complete the test and submit the test
- Click on Get Diff
- The user should be able to view their test result as well as the per word difference between the master transcript and their transcript.

### Intention:

- allow the user to see their transcript data and diff when they give practice Test instead of only showing the accuracy and Diff
